commit a7ebecb9fbee2190e649c44ed53f1299013baa30
Author: Andreas Sturmlechner <andreas.sturmlechner@gmail.com>
Date:   Sun Mar 18 11:56:57 2018 +0100

    Fix build with Qt 5.11 (missing headers)
    
    Reviewers: #calligra:_3.0, anthonyfieroni, danders
    
    Reviewed By: #calligra:_3.0, anthonyfieroni, danders
    
    Subscribers: anthonyfieroni, danders
    
    Tags: #calligra:_3.0
    
    Differential Revision: https://phabricator.kde.org/D11454

diff --git a/libs/widgets/KoCsvImportDialog.cpp b/libs/widgets/KoCsvImportDialog.cpp
index 0ffdcf62a70..cdca006dc62 100644
--- a/libs/widgets/KoCsvImportDialog.cpp
+++ b/libs/widgets/KoCsvImportDialog.cpp
@@ -21,6 +21,7 @@
 #include "KoCsvImportDialog.h"
 
 // Qt
+#include <QButtonGroup>
 #include <QTextCodec>
 #include <QTextStream>
 
diff --git a/libs/widgets/KoPageLayoutWidget.cpp b/libs/widgets/KoPageLayoutWidget.cpp
index f91555ca262..a3816f96d8a 100644
--- a/libs/widgets/KoPageLayoutWidget.cpp
+++ b/libs/widgets/KoPageLayoutWidget.cpp
@@ -23,6 +23,8 @@
 
 #include <KoUnit.h>
 
+#include <QButtonGroup>
+
 class Q_DECL_HIDDEN KoPageLayoutWidget::Private
 {
 public:
diff --git a/plugins/chartshape/dialogs/TableEditorDialog.cpp b/plugins/chartshape/dialogs/TableEditorDialog.cpp
index c0d5136f09e..d2a772e83c2 100644
--- a/plugins/chartshape/dialogs/TableEditorDialog.cpp
+++ b/plugins/chartshape/dialogs/TableEditorDialog.cpp
@@ -24,6 +24,7 @@
 
 // Qt
 #include <QAbstractItemModel>
+#include <QAction>
 
 // Calligra
 #include <KoIcon.h>
diff --git a/plugins/formulashape/FormulaToolWidget.cpp b/plugins/formulashape/FormulaToolWidget.cpp
index ed109192cc5..8f521779617 100644
--- a/plugins/formulashape/FormulaToolWidget.cpp
+++ b/plugins/formulashape/FormulaToolWidget.cpp
@@ -30,6 +30,7 @@
 #include <QWidgetAction>
 #include <QTableWidget>
 #include <QAction>
+#include <QHeaderView>
 #include <QMenu>
 
 FormulaToolWidget::FormulaToolWidget( KoFormulaTool* tool, QWidget* parent )
diff --git a/sheets/dialogs/LayoutDialog.cpp b/sheets/dialogs/LayoutDialog.cpp
index a0a9832ddfc..7d7db53823b 100644
--- a/sheets/dialogs/LayoutDialog.cpp
+++ b/sheets/dialogs/LayoutDialog.cpp
@@ -36,6 +36,7 @@
 #include <math.h>
 
 #include <QIntValidator>
+#include <QButtonGroup>
 #include <QCheckBox>
 #include <QFrame>
 #include <QLabel>
diff --git a/words/part/dialogs/KWAnchoringProperties.cpp b/words/part/dialogs/KWAnchoringProperties.cpp
index d64208cbdb6..bfddb3a03f5 100644
--- a/words/part/dialogs/KWAnchoringProperties.cpp
+++ b/words/part/dialogs/KWAnchoringProperties.cpp
@@ -35,6 +35,7 @@
 
 #include <kundo2command.h>
 
+#include <QButtonGroup>
 #include <QComboBox>
 
 const int KWAnchoringProperties::vertRels[4][20] = {
diff --git a/words/part/dialogs/KWRunAroundProperties.cpp b/words/part/dialogs/KWRunAroundProperties.cpp
index e38599a06d7..7e8b2d51ef9 100644
--- a/words/part/dialogs/KWRunAroundProperties.cpp
+++ b/words/part/dialogs/KWRunAroundProperties.cpp
@@ -28,6 +28,8 @@
 
 #include <kundo2command.h>
 
+#include <QButtonGroup>
+
 KWRunAroundProperties::KWRunAroundProperties(FrameConfigSharedState *state)
         : m_state(state)
 {
From ee83e0f2c251072e47a2799619cdc79efe67e651 Mon Sep 17 00:00:00 2001
From: David Faure <faure@kde.org>
Date: Tue, 3 Apr 2018 00:31:19 +0200
Subject: [PATCH 03/10] Fix compilation with Qt 5.11 (missing include)

---
 stage/part/KPrPresentationTool.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stage/part/KPrPresentationTool.cpp b/stage/part/KPrPresentationTool.cpp
index ae743da8fe0..3007f9101e4 100644
--- a/stage/part/KPrPresentationTool.cpp
+++ b/stage/part/KPrPresentationTool.cpp
@@ -32,6 +32,7 @@
 #include <QDesktopServices>
 #include <QUrl>
 #include <QDBusConnection>
+#include <QFrame>
 
 #include <KoShape.h>
 #include <KoShapeManager.h>
-- 
2.17.0

